docs/auroLabsMinors.md

```diff
@@ -0,0 +1,173 @@
+  <auro-accordion id="newWork" chromeless noProfile>
+    <span slot="trigger" class="trigger">1. Install WC-Generator</span>
+    <div class="lightText">
+      <p>Install the <auro-hyperlink href="https://auro.alaskaair.com/getting-started/developers/generator/install">WC-Generator</auro-hyperlink>, if not already installed</p>
+      <pre class="pre">  $ npm i @alaskaairux/wc-generator -g</pre>
+    </div>
+  </auro-accordion>
+  <auro-accordion id="newWork" chromeless noProfile>
+    <span slot="trigger" class="trigger">2. Create your new custom element repository locally</span>
+    <div class="lightText">
+      <p>Once the generator is installed, run the generator command to create a new custom element repository.</p>
+      <pre class="pre">  $ wc-generate --npm @aurolabs --name [element name]</pre>
+    </div>
+  </auro-accordion>
+  <auro-accordion id="newWork" chromeless noProfile>
+    <span slot="trigger" class="trigger">3. Change into new directory</span>
+    <div class="lightText">
+      <p>You need to be inside the newly created directory in order to complete the following steps.</p>
+    </div>
+  </auro-accordion>
+  <auro-accordion id="newWork" chromeless noProfile>
+    <span slot="trigger" class="trigger">4. Run tests</span>
+    <div class="lightText">
+      <p>Run the tests to ensure that all things are working as expected.</p>
+      <pre class="pre">  $ npm run test</pre>
+    </div>
+  </auro-accordion>
+  <auro-accordion id="newWork" chromeless noProfile>
+    <span slot="trigger" class="trigger">5. Start the server</span>
+    <div class="lightText">
+      <p>Start the server to make sure that all dependencies are set up correctly.</p>
+      <pre class="pre">  $ npm run serve</pre>
+    </div>
+  </auro-accordion>
+  <auro-accordion id="newWork" chromeless noProfile>
+    <span slot="trigger" class="trigger">6. Create an internal Github repo</span>
+    <div class="lightText">
+      <p>For Alaska employees only. Create a new <code>internal</code> repository with <auro-hyperlink target="_blank" href="https://github.com/AlaskaAirlines">github.com/AlaskaAirlines</auro-hyperlink>. Once completed, push your <code>main</code> branch to the remote repository.</p>
+      <p>Please be sure to name the repo with <strong>auro</strong> to properly follow our open source repository guidelines. E.g. auro-[your component name].
+      <p><strong>Randos will be deleted!</strong></p>
+    </div>
+  </auro-accordion>
+</auro-accordion-group>
+
+## Getting to work
+
+Now that your install is setup and your repo is ready to go, let's get to work!
+
+<auro-accordion-group>
+  <auro-accordion id="newWork" chromeless noProfile>
+    <span slot="trigger" class="trigger">1. Create your new feature branch</span>
+    <div class="lightText">
+      <p>In your local repo, create a new working branch called <code>initial-build</code>, this will be the branch you will compare to <code>main</code> for the pull request later.</p>
```

Comment:
```suggestion
      <p>In your local repo, create a new working branch called <code>initial-build</code>. This will be the branch you will compare to <code>main</code> for the pull request later.</p>
```